.zenodo.json

```diff
@@ -2,6 +2,12 @@
   "title": "fMRIPrep: a robust preprocessing pipeline for functional MRI",
   "description": "<p>fMRIPrep is a robust and easy-to-use pipeline for preprocessing of diverse fMRI data. The transparent workflow dispenses of manual intervention, thereby ensuring the reproducibility of the results.</p>",
   "contributors": [
+    {
+      "affiliation": "Speech & Hearing Bioscience & Technology Program, Harvard University",
+      "name": "Mentch, Jeff",
+      "orcid": "0000-0002-7762-8678",
+      "type": "Researcher"
+    },
```

Comment:
This file is updated automatically. Please add your name to [`.maint/contributors.json`](https://github.com/nipreps/fmriprep/blob/c707938e1462e77b1c3a1216a855ceb0aba70845/.maint/contributors.json) instead. Contributors are ordered alphabetically.